src/foam/nanos/menu/SequenceMenu.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.menu',
+  name: 'SequenceMenu',
+  extends: 'foam.nanos.menu.AbstractMenu',
+
+  requires: [
+    'foam.util.async.Sequence'
+  ],
+
+  properties: [
+    {
+      class: 'FObjectArray',
+      // of: 'foam.util.FluentSpec',
```

Comment:
So whats the intent here saying, if Sequence is a subclass of Fluent, shouldn't this be only Sequences instead of FluentSpecs? Also what is the difference between a Sequence and Fluent. Is a Sequence a contextAgent aware Fluent essentially? Would help if you added documentation to foam.util.async.Sequence.

Reply:
`Fluent` is the interface used by `Sequence` and `Element` (chainable `.add()` and similar methods). `FluentSpec` allows defining one of these in a journal.

Reply:
Im a bit confused by this... so unresolving for more info, plz.
The Sequence - which takes a list of Objects and executes them...with other functionality...
The Sequence extends Fluent.
Fluents are A domain-specific "language" - a model with chainable methods...https://en.wikipedia.org/wiki/Fluent_interface
Then this PR creats a FluentSpec(Interface), AddFluentSpec(implementation of FluentSpec), FObjectSpec(journable FObject - used in AddFluentSpec) and also created (FObjectArraySpec ...never used, not sure what would use this)

The Object that is being executed in the sequence is not a Fluent [(code)](https://github.com/kgrgreer/foam3/blob/9423f60a2be3cf299bd23cca49188dc3ccd17a93/src/foam/util/async/Sequence.js#L27-L29)- the Sequence is - but the sequences of things are just FObjects from what I see. Thus naming the executable objects FluentSpecs, don't seem to add up in my mind.

Also wonder why the 'of' property property is commented out here?

Reply:
The purpose of `FluentSpec` is to make it possible to write the same information you'd write using the fluent interface in a journal file.

For example:
```
mySequence
  .add("something")
  .remove("something")
```

would become something like:
```
[
  { class: "AddFluentSpec", spec: "something" },
  { class: "RemoveFluentSpec", spec: "something" }
]
```

(this is simplified as `"something"` wouldn't be a valid spec)

`FObjectArraySpec` is a byproduct of another idea I was testing (not having FluentSpec and instead using FObjectArraySpec). It's not used anymore; I left it in the PR so it doesn't need to be rewritten if it's used later. Right now there's no way to store an array of FObject specs in a journal without this (that I know of)

The `of` property is commented out because interfaces are invalid as the `of` for an FObjectProperty, so FOAM wouldn't let me do this. However, the `of` really should be `FluentSpec`.